server/src/internalClusterTest/java/org/opensearch/action/admin/indices/create/RemoteCloneIndexIT.java

```diff
@@ -60,6 +61,7 @@
 import static org.opensearch.test.hamcrest.OpenSearchAssertions.assertHitCount;
 import static org.hamcrest.Matchers.equalTo;
 
+@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, numDataNodes = 0)
```

Comment:
Why do we need to change the scope?

Reply:
One of the test of this class need to use `reloadable-fs-repo` repo so for that test we need to set `asyncUploadMockFs` to false. Hence we need scope of this class on test level.